tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -192,9 +192,13 @@ After logging in you should see the `HelloWorld` servlet which is now served by
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 3: ](Protect your backend microservice)]
-After authentication works with the App Router, your java backend service is still fully visible in the web and not protected. Therefore, we need to protect our java microservices as well so that they accept requests with valid `JWTs` for the current user only. In addition, we will setup the microservice in a way that it deals with authorization, i.e., understands the OAuth scopes from the JWT that we have configured previously using the `xs-security.json` file.
+After authentication works with the App Router, your java backend service is still fully visible in the web and not protected. Therefore, we need to protect our java microservices as well so that they accept requests with valid JWTs for the current user only. In addition, we will setup the microservice in a way that it deals with authorization, i.e., understands the OAuth scopes from the JWT that we have configured previously using the `xs-security.json` file.
```

Comment:
Please use You and not the royal We (unless, in rare cases, you are referring to SAP).

You should rather say: "...Therefore, you need to protect our java...".

Reply:
Done